src/patternfly/components/Chip/examples/chip-example.hbs

```diff
@@ -23,14 +23,21 @@
     Chip
   {{/chip-text}}
   {{#> badge badge--modifier="pf-m-read"}}
-  7
-{{/badge}}
+    7
+  {{/badge}}
   {{#> button button--modifier="pf-m-plain" button--attribute='aria-labelledby="remove_chip_three chip_three" aria-label="Remove" id="remove_chip_three"'}}
     {{> button-icon button-icon--type="times-circle"}}
   {{/button}}
 {{/chip}}
 <br>
 <br>
+{{#> chip chip--type="div" chip--modifier="pf-m-read-only"}}
+  {{#> chip-text}}
```

Comment:
should this chip-text have an id

Reply:
I don't think so. The `id` is only used with `aria-labelledby` when there is an icon button (the close button)